src/model.jl

```diff
@@ -3,7 +3,7 @@ export find_solution, find_search_interval
 """
     (a, b) = find_search_interval(f)
 
-Returns an interval such that ``f(a) × f(b) ≯ 0``.
+Returns an interval such that ``f(a) × f(b) > 0``.
```

Comment:
```suggestion
Returns an interval such that ``f(a) × f(b) ≤ 0``.
```
This was meant as "not positive". Instead, we can change to <= to make it clearer.